### DISCUSSION TITLE
【session 5】KZG10多项式承诺相关疑问

### DISCUSSION THREAD

**Discussion by Howard-Hu** [zkp-co-learning/zkp-co-learn#53] [Q&A（提问题在隔壁~）] [answered]
1. setup相关的问题:
- 如何安全的生成秘密值，并且用它生成公开参数;有没有相关详细资料推荐;
- 公开参数的长度一般设定多少，是不是只需要满足大于需要承诺的多项式的次数+1就可以?

2.   $\alpha$ 相关的问题:
- 引入  $\alpha$ 的动机和作用是什么;

3. KZG10是不是零知识的?如果因为它是确定的而不是零知识性，那么在Fiat-Shamir使用哈希函数的输出作为”随机数“也使得其变成了确定性的，那么Fiat-Shamir变换岂不是有问题?

**Comment by Demian101** [comment, marked as answer]
第三次 Office Hour 上由@郭宇 老师讲解
